other/block-cluster-admin-from-ns/02-setup-user.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- command: openssl genrsa -out testuser.key 2048
+- command: openssl req -new -key testuser.key -out testuser.csr -subj "/CN=testuser/O=testorg"
+- command: "docker cp kind-control-plane:/etc/kubernetes/pki/ca.crt ."
+- command: "docker cp kind-control-plane:/etc/kubernetes/pki/ca.key ."
```

Comment:
These are fine, but I still prefer to eliminate as many tooling dependencies as we can, especially those which we're assuming to be on these runners. That's why in the script I gave you I read this info from the kubeconfig where it is already provided.

Reply:
Have changed to use the script now.